src/components/card-collapse/index.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Collapse } from 'reactstrap';
+import Button from '../button';
+import Card from '../card';
+import CardBody from '../card-body';
+import CardHeader from '../card-header';
+
+class CardCollapse extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      isOpen: props.isOpen,
+    };
+
+    this.toggle = this.toggle.bind(this);
+  }
+
+  toggle() {
+    this.setState({ isOpen: !this.state.isOpen });
+  }
+
+  render() {
+
+    const {
+      cardCollapseTitle,
+      cardHeaderClassName,
+      children,
+      className,
+      isStacked,
+    } = this.props;
+
+    const cardClasses = `card-collapse ${isStacked ? 'mb-0' : ''} ${className}`;
+
+    return (
+      <Card className={cardClasses}>
+        {cardCollapseTitle &&
```

Comment:
i'd probably just call this title, and make it either required, or have a default. if it is missing, the whole header/button doesnt render and then there is no way to open/close the collapse. not totally sure i like that behavior but i also cant think of a time when you would want this without a header so i dont think its that important.

Reply:
I renamed prop to title and made it so the button would display even if title was empty